internal/services/mobilenetwork/mobile_network_packet_core_control_plane_resource.go

```diff
@@ -0,0 +1,687 @@
+				string(packetcorecontrolplane.BillingSkuGFive),
+				string(packetcorecontrolplane.BillingSkuGOneZero),
+			}, false),
+		},
+
+		"user_equipment_mtu_in_bytes": {
+			Type:         pluginsdk.TypeInt,
+			Optional:     true,
+			Default:      1440,
+			ValidateFunc: validation.IntBetween(1280, 1930),
+		},
+
+		"core_network_technology": {
+			Type:     pluginsdk.TypeString,
+			Optional: true,
+			ValidateFunc: validation.StringInSlice([]string{
+				string(packetcorecontrolplane.CoreNetworkTypeFiveGC),
+				string(packetcorecontrolplane.CoreNetworkTypeEPC),
+			}, false),
+		},
+
+		"platform": {
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			MaxItems: 1,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"edge_device_id": {
+						Type:             pluginsdk.TypeString,
+						Optional:         true,
+						DiffSuppressFunc: suppress.CaseDifference,
+						AtLeastOneOf:     []string{"platform.0.edge_device_id", "platform.0.azure_stack_hci_cluster_id", "platform.0.azure_arc_connected_cluster_id", "platform.0.custom_location_id"},
+
+						ValidateFunc: func(i interface{}, s string) ([]string, []error) {
+							// a workaround, sometimes the id from user is with `DataBoxEdgeDevices` instead of `dataBoxEdgeDevices`.
+							// while the service accepts both format.
+							if v, ok := i.(string); ok {
+								i = strings.ReplaceAll(v, "DataBoxEdgeDevices", "dataBoxEdgeDevices")
+							}
+							return devices.ValidateDataBoxEdgeDeviceID(i, s)
+						},
+					},
+
+					"azure_stack_hci_cluster_id": {
+						Type:         pluginsdk.TypeString,
+						Optional:     true,
+						ValidateFunc: validation.StringIsNotEmpty,
+						AtLeastOneOf: []string{"platform.0.edge_device_id", "platform.0.azure_stack_hci_cluster_id", "platform.0.azure_arc_connected_cluster_id", "platform.0.custom_location_id"},
+					},
+
+					"azure_arc_connected_cluster_id": {
+						Type:         pluginsdk.TypeString,
+						Optional:     true,
+						ValidateFunc: validation.StringIsNotEmpty,
+						AtLeastOneOf: []string{"platform.0.edge_device_id", "platform.0.azure_stack_hci_cluster_id", "platform.0.azure_arc_connected_cluster_id", "platform.0.custom_location_id"},
+					},
+
+					"custom_location_id": {
+						Type:         pluginsdk.TypeString,
+						Optional:     true,
+						ValidateFunc: validation.StringIsNotEmpty,
+						AtLeastOneOf: []string{"platform.0.edge_device_id", "platform.0.azure_stack_hci_cluster_id", "platform.0.azure_arc_connected_cluster_id", "platform.0.custom_location_id"},
+					},
+
+					"type": {
+						Type:     pluginsdk.TypeString,
+						Required: true,
+						ValidateFunc: validation.StringInSlice([]string{
+							string(packetcorecontrolplane.PlatformTypeAKSNegativeHCI),
+							string(packetcorecontrolplane.PlatformTypeThreePNegativeAZURENegativeSTACKNegativeHCI),
+							string("BaseVM"), // a workaround for there might be existing one with this value. allow user to import them.
```

Comment:
I have talked to svc team before. They removed it from Swagger because it's going to be deprecated but there might be some existing ones created by older API version.. 
Maybe just removing it is a better choice?